src/views/CollectionIndex/Samples/Page.tsx

```diff
@@ -49,49 +50,58 @@ export const Page = ({ data, cart, collectionId, history }: IPage) => {
       <div>
         <div className="wrapper-header">Samples</div>
         <div className="wrapper-img">
-          {data.products.edges.map(
-            ({ node: { name, id, pricing, thumbnail, variants } }, idx) => (
-              <div className="wrapper-img-main" key={idx}>
-                <div
-                  className={
-                    samples
-                      ? samples.indexOf(variants[0].id) > -1
-                        ? "wrapper-img-main-inner withBorder"
-                        : "wrapper-img-main-inner noBorder"
-                      : ""
-                  }
-                >
-                  <div className="wrapper-img-main-inner--header">
-                    <span>{name}</span>
-                  </div>
-                  <div className="wrapper-img-main-inner--img">
-                    <img
-                      src={thumbnail.url}
-                      onClick={() => selectSample(variants[0].id)}
-                      id={id}
-                      key={idx}
-                    />
-                  </div>
-                  <div className="wrapper-img-main-inner--price">
-                    <span className="old-price">
-                      <del>${pricing.priceRange.start.gross.amount}</del>
-                    </span>
-                    <span className="new-price">
-                      ${pricing.priceRange.start.net.amount}
-                    </span>
+          {sampleCount > 0 ? (
```

Comment:
You can rename `sampleCount` to `samplesExist`, and just use `samplesExist ? (...) : (...)`

Reply:
Done